classes/class-dintero-checkout-confirmation.php

```diff
@@ -92,6 +92,13 @@ public function handle_success( $transaction_id, $order ) {
 			}
 		}
 
+		// Save shipping id to the order.
+		$shipping = $order->get_shipping_methods();
+		if ( ! empty( $shipping ) ) {
+			$shipping_option_id = $dintero_order['shipping_option']['id'] ?? reset( $shipping );
```

Comment:
what happens if `$dintero_order['shipping_option']` is `null`?

Reply:
We'll default to the shipping option that we retrieved from the WC order.